### DISCUSSION TITLE
Cooler borgs

### DISCUSSION THREAD

**Discussion by mirrorcult** [space-wizards/space-station-14#3989] [Feature Request]
https://hackmd.io/@rQ2pYSFtRl20YDfmkEtgzA/H1ga1vqvO

tldr
- more modular borg exosuits
- better borg modules
- borgs can't talk by default without a speech module
- borgs aren't asimov, instead just have one law to listen to whatever the AI says, if the AI dies then they have no masters

**Comment by hamurlik** [comment]
cool based red pilled

**Comment by CatMagic1** [comment]
>become cool engie borg
>do cool engie borg things
>beat someone to death for no reason
>ai demotes me to cleanbot?
